packages/config/src/index.ts

```diff
@@ -71,6 +71,7 @@ export const types = Object.assign({
   offline: Boolean,
   'package-import-method': ['auto', 'hardlink', 'clone', 'copy'],
   pnpmfile: String,
+  'pnpm-bin': String,
```

Comment:
might be better to name it `global-bin-dir`

Reply:
Changed.